src/components/ObjectArrayEditor.vue

```diff
@@ -18,8 +18,10 @@
             <CustomFormItem :prop="getProp($index, key)" :rules="getFormItemRules(key)">
               <SchemaFormItem
                 v-model="displayTableData[$index][key]"
-                :type="(value.type as any)"
-                :symbols="(value.symbols as string[] | number[] | undefined)"
+                :type="value.type"
+                :format="value.format"
+                :symbols="value.symbols"
```

Comment:
🙏🙏🙏🙏

Reply:
what happened? 😂

Reply:
You've solved some problems that I was supposed to handle.